packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -118,6 +119,7 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> {
     let isRTL = getRTL();
     let isOnRightSide = isRTL ? isLeft : !isLeft;
     const headerTextId = id + '-headerText';
+    const customWidthStyles = (type === PanelType.custom) ? { width: customWidth } : {};
```

Comment:
Does this work on smaller screens? E.g. If the width is 888px, what happens on a 320px wide device?